packages/code/src/CopyButton/CopyButton.tsx

```diff
@@ -39,7 +39,7 @@ function CopyButton({ onCopy, contents, className, ...rest }: CopyProps) {
   const { portalContainer } = usePopoverPortalContainer();
   const { showPanel, isLoading } = useCodeContext();
 
-  const lgids = getLgIds();
+  const lgIds = getLgIds();
```

Comment:
If a custom `dataLgId` is passed into `Code` should this be inheriting that value?